libs/contact/tsconfig.json

```diff
@@ -23,9 +23,21 @@
       "@daffodil/contact": [
```

Comment:
The `dist` paths can all be replaced with a wildcard path alias.

Reply:
Fixed in 061de9d8ecfc2da4c3df5910d597a21954e70e76